esquery.js

```diff
@@ -200,6 +198,7 @@ function sibling(node, selector, ancestry, side) {
     keys = estraverse.VisitorKeys[parent.type];
     for (i = 0, l = keys.length; i < l; ++i) {
         listProp = parent[keys[i]];
+        // istanbul ignore else
```

Comment:
These last two remaining ones should be able to be covered by passing a malformed AST node (in the `parents`) that has a non-array where an array would be expected. After that, we should be good to merge this PR.

Reply:
I think I was able to get this with an ok AST node. Wasn't sure the best name for the test though.